templates/notebook_instance/sdk_read_one_post_set_output.go.tpl

```diff
@@ -0,0 +1,10 @@
+// NotebookInstanceLifecycleConfigName does not get generated by the code generator.
```

Comment:
why?
add this:
because the field name in spec vs Describe response do not match

Reply:
added it